### DISCUSSION TITLE
Polkit authenticator floating window rule

### DISCUSSION THREAD

**Discussion by jimmy-979** [hyprwm/Hyprland#1785] [Q&A]
Hi! I installed polkit-kde-agent for my authentication agent as specified in the wiki. I was wandering if there is any window rule to make the password prompt window float.

**Comment by vaxerski** [comment]
![image](https://user-images.githubusercontent.com/43317083/225168276-8a6525f9-7a42-41c1-a280-7aa3c163d403.png)
https://wiki.hyprland.org/0.23.0beta/Configuring/Window-Rules/

**Reply by jimmy-979** [reply]
I tried `windowrule = float, ^(polkit-kde-agent)$` and it didn't seem to work. Maybe the polkit password prompt application has another name?

**Reply by Sohail-XD** [reply]
You missed the syntax. For class( hyprctl clients in terminal to know class) `windowrule=float,org.kde.polkit-kde-authentication-agent-1 ` .
